bin/p2-exec/main.go

```diff
@@ -80,20 +81,55 @@ func main() {
 		}
 	}
 
+	cgroupPath := ""
+	if *podID != "" {
+		cgConfig, err := cgGetConfig("RESOURCE_LIMITS", "")
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		node, err := os.Hostname()
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		cg, err := cgroups.Find()
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		err = os.MkdirAll(filepath.Join(cg.CPU, "p2", node), 0755)
```

Comment:
this seems like something that belongs in the `cgroups` package so we can reuse it for other things